source/transfer_list.rst

```diff
@@ -66,12 +67,13 @@ changes will be backwards-compatible to older readers.
    * - signature
      - 0x4
      - 0x0
-     - The value of signature must be `0x6e_d0ff`.
+     - The value of signature must be `0x40_b10b`.
```

Comment:
Wasn't part of the reason for changing the signature that it makes more sense to use all 4 bytes? Now we're again leaving the top byte 0 which seems like a waste? I'm aware that this is also used in the 3-byte field of the register convention, but I still don't think that's a good reason to leave it 0x00 here. Or if we do decide that 3 bytes are enough we might as well use the remaining byte for something useful instead, like the flags (or maybe the version field should be moved here which would make it very similar to how the top byte is used in the register convention).

Reply:
Sorry, I don't understand what you mean. Didn't we agree on the solution that increases the TL header size by 8 bytes now? Isn't that what this patch is doing?

As far as I can tell the signature field here still has 4 bytes, it's just that one of them is (still) 0 and I think it could be something more distinctive.